src/main.cpp

```diff
@@ -354,6 +354,18 @@ int main(int argc, char *argv[])
     }
 
     setupDebug();
+
+#if (defined _WIN32 || defined WINDOWS)
+    // Use the ANSI code page 1252 to work around some language output bugs.
+    if (setlocale(LC_ALL, ".1252") == NULL) {
+        DebugLog(D_WARNING, D_MAIN) << "Error while setlocale(LC_ALL, '.1252').";
+    }
+#else
+    if (setlocale(LC_ALL, "") == NULL) {
+        DebugLog(D_WARNING, D_MAIN) << "Error while setlocale(LC_ALL, '').";
+    }
+#endif
```

Comment:
My proposal here would be to just call `setlocale(LC_ALL, "")` (with the `NULL` check as-is) without all the guards.

Then the `setlocale(LC_ALL, ".1252")` (also with the `NULL` check as-is) could be shoved back to translations.cpp and run only for `LOCALIZE=1` on Windows platforms. I'd let it be there due to that comment:

> Use the ANSI code page 1252 to work around some language output bugs.

No idea what 'bugs' there might be, but it seems better to stay on the 'safe(er) side' of things...